_blueTheme.scss

```diff
@@ -523,4 +523,10 @@
             }
         }
     }
+
+    /* Mat Nav List */
+    .mat-list-base .blui-info-list-item .mat-list-item-disabled {
```

Comment:
Is it possible to move this in `common.scss` ?

Reply:
It's possible but the `common.scss` file has historically been used for all non-color overrides.  I wouldn't want to start spreading where our colors are located.